modules/ztp-post-install-hub-factory-pipeline-checks.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="post-hub-factory-pipeline-verification-checks_{context}"]
+= Post hub factory pipeline verification checks
+
+Perform the following steps after completion of the hub factory pipeline run.
+
+.Prerequisites
+
+* An {product-title} hub cluster
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Verify ACM is successfully installed.
++
+[source,terminal]
+----
+oc get pod -n open-cluster management
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                                        READY   STATUS      RESTART         AGE
+application-chart-ee7d2-applicastionui-7d99756554-jrs24     1/1     RUNNING     0               6m31s
+application-chart-ee7d2-applicastionui-7d99756554-jrs24     1/1     RUNNING     0               6m31s
+application-chart-ee7d2-applicastionui-7d99756554-jrs24     1/1     RUNNING     0               6m31s
+application-chart-ee7d2-applicastionui-7d99756554-jrs24     1/1     RUNNING     0               6m31s
+assisted-image-service-67489b657b-68qtg                     1/1     RUNNING     0               2m30s
+assisted-service-5b8874ffd9-rjrg                            2/2     RUNNING     1 (2m19s ago)   2m30s
+----
+
+. Verify the HTTPD server is successfully running.
++
+[source,terminal]
+----
+$ oc get pod -n default
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                       READY   STATUS      RESTART         AGE
+httpd-5479bfd6cb-2p1d4     1/1     RUNNING     0               150m
+----
+
+. Verify the internal registry is running:
++
+[source,terminal]
+----
+$ oc get pod -n kubeframe-registry
```

Comment:
@jparrill @iranzo 
the naming `kubeframe` hasn't change yet ?

Reply:
yup, it should :)

Reply:
```suggestion
$ oc get pod -n  ztpfw-registry
```